modules/super-admin/components/product-listing/ProductListingTable.tsx

```diff
@@ -13,17 +13,27 @@ export const LoadingTable = () => {
   );
 };
 
-const ProductListingTable = ({ data, isLoading }: { data: any; isLoading: boolean }) => {
+const ProductListingTable = ({
+  data,
+  isLoading,
+  currentPage,
+  setCurrentPage,
+}: {
+  data: any;
+  isLoading: boolean;
+  currentPage: number;
+  setCurrentPage: any;
```

Comment:
react state updater has a type, please google and use it to replace any for setCurrentPage